app/src/main/java/com/github/se/studybuddies/ui/video_call/CallLobbyScreen.kt

```diff
@@ -41,8 +46,24 @@ fun CallLobbyScreen(
 ) {
   LockScreenOrientation(orientation = Configuration.ORIENTATION_PORTRAIT)
   val call by remember { mutableStateOf(callLobbyViewModel.call) }
+  val isLoading by callLobbyViewModel.isLoading.collectAsState()
   val isCameraEnabled by call.camera.isEnabled.collectAsState()
   val isMicrophoneEnabled by call.microphone.isEnabled.collectAsState()
+  val context = LocalContext.current
+
+  LaunchCallPermissions(
+      call = call,
+      onPermissionsResult = {
+        if (it.values.contains(false)) {
+          Toast.makeText(
```

Comment:
I like that you displayed a Toast if the permissions are not granted, but is it possible to ask again because if it was a missclick the user should go to the android settings to manually allow the permissions to be able to use the videocall function

Reply:
If u go to the call lobby again after denying the permissions, it should ask it again. Thats what happened when I tested it at least